hebi_a-2084-01_moveit_config/config/A-2084-01.srdf

```diff
@@ -10,35 +10,42 @@
+        <joint name="Arm/J4_wrist1/X5_1" />
     </group>
     <!--GROUP STATES: Purpose: Define a named state for a particular group, in terms of joint values. This is useful to define states like 'folded arms'-->
     <group_state name="home" group="hebi_arm">
-        <joint name="HEBI/base/X5_4" value="-1" />
-        <joint name="HEBI/elbow/X5_4" value="-2" />
-        <joint name="HEBI/shoulder/X5_9" value="0" />
-        <joint name="HEBI/wrist1/X5_1" value="0" />
+        <joint name="Arm/J1_base/X5_4" value="-1" />
+        <joint name="Arm/J2_shoulder/X5_9" value="0" />
+        <joint name="Arm/J3_elbow/X5_4" value="-2" />
+        <joint name="Arm/J4_wrist1/X5_1" value="0" />
     </group_state>
     <!--VIRTUAL JOINT: Purpose: this element defines a virtual joint between a robot link and an external frame of reference (considered fixed with respect to the robot)-->
     <virtual_joint name="virtual_joint" type="fixed" parent_frame="world" child_link="world" />
     <!--DISABLE COLLISIONS: By default it is assumed that any link of the robot could potentially come into collision with any other link in the robot. This tag disables collision checking between a specified pair of links. -->
-    <disable_collisions link1="HEBI/base/INPUT_INTERFACE" link2="HEBI/elbow/INPUT_INTERFACE" reason="Never" />
-    <disable_collisions link1="HEBI/base/INPUT_INTERFACE" link2="HEBI/shoulder/INPUT_INTERFACE" reason="Never" />
-    <disable_collisions link1="HEBI/base/INPUT_INTERFACE" link2="shoulder_bracket/INPUT_INTERFACE" reason="Adjacent" />
-    <disable_collisions link1="HEBI/elbow/INPUT_INTERFACE" link2="HEBI/shoulder/INPUT_INTERFACE" reason="Never" />
-    <disable_collisions link1="HEBI/elbow/INPUT_INTERFACE" link2="HEBI/wrist1/INPUT_INTERFACE" reason="Never" />
-    <disable_collisions link1="HEBI/elbow/INPUT_INTERFACE" link2="elbow_wrist1/INPUT_INTERFACE" reason="Adjacent" />
-    <disable_collisions link1="HEBI/elbow/INPUT_INTERFACE" link2="shoulder_bracket/INPUT_INTERFACE" reason="Never" />
-    <disable_collisions link1="HEBI/elbow/INPUT_INTERFACE" link2="shoulder_elbow/INPUT_INTERFACE" reason="Adjacent" />
-    <disable_collisions link1="HEBI/shoulder/INPUT_INTERFACE" link2="HEBI/wrist1/INPUT_INTERFACE" reason="Never" />
-    <disable_collisions link1="HEBI/shoulder/INPUT_INTERFACE" link2="shoulder_bracket/INPUT_INTERFACE" reason="Adjacent" />
-    <disable_collisions link1="HEBI/shoulder/INPUT_INTERFACE" link2="shoulder_elbow/INPUT_INTERFACE" reason="Adjacent" />
-    <disable_collisions link1="HEBI/wrist1/INPUT_INTERFACE" link2="elbow_wrist1/INPUT_INTERFACE" reason="Adjacent" />
-    <disable_collisions link1="HEBI/wrist1/INPUT_INTERFACE" link2="shoulder_bracket/INPUT_INTERFACE" reason="Never" />
-    <disable_collisions link1="HEBI/wrist1/INPUT_INTERFACE" link2="shoulder_elbow/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="Arm/J1_base/INPUT_INTERFACE" link2="Arm/J2_shoulder/INPUT_INTERFACE" reason="Never" />
```

Comment:
There are a number of `end_effector` links added here, but this is the non-gripper config.  Are these referring to the "null" end effector, or did something get added accidentally?